src/components/BSection/BSection.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <section>
+    <span class="sides"></span>
```

Comment:
Shouldn't this be solved by having some sort of grid? Adding the sides to a component would seem to take away a lot of flexibility.

Reply:
We could implement CSS Grid.  It has less support than flex-box.

https://caniuse.com/#search=grid

Right now this component just needs text and some left/right padding.  We don't really have a use case for anything beyond that yet.